spec.html

```diff
@@ -6817,6 +6817,8 @@ <h1>
       <dl class="header">
       </dl>
       <emu-alg>
+        1. If the host is a web browser, then
```

Comment:
This is the first time that I see something like this in an algo step, I have mix feelings about it.

Reply:
@phoddie requested that this be Annex B only, so that non-browser hosts wouldn't have to read it at all. But I really do not want to put any more things in Annex B. So this was the compromise.

Reply:
While I do think it seems weird, and I think it's far more appropriate to have the check inside the host hook, this is "fine" imo - and I believe this represents pushback from @phoddie (who may want to clarify)

Reply:
I must agreed that having this in the algos rather than in a separate sections that nobody looks at is a lot better, I was just hesitant because this is the first time I have seen this condition in the spec.